### PR TITLE
move ubuntu demo to standalone folder under gcp and add GCP tutorial …

### PR DESCRIPTION
**Pull request by youf05**
- Updated folder structure to move GCP Ubuntu demo into subfolder
- Added screenshots to help new GCP users create project, enable API access, and create account key

### CODE REVIEW

File: azure_arc_servers_jumpstart/gcp/windows/main.tf [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+// An Azure Resource Group
+resource "azurerm_resource_group" "azure_rg" {
+  name = var.azure_resource_group
+  location = var.azure_location
+}
+
+// A single Google Cloud Engine instance
+resource "google_compute_instance" "default" {
+ name = "azure-arc-gcp-demo-${random_id.instance_id.hex}"
```
Comment (likamrat) [inline azure_arc_servers_jumpstart/gcp/windows/main.tf:14]: I think the name is too long. Also, consider the 15 characters' Windows hostname limit. Can drop the "azure" IMO.

File: azure_arc_servers_jumpstart/gcp/windows/main.tf [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    location = var.azure_location
+    }
+  )
+  filename = "scripts/install_arc_agent.sh"
+}
+
+// A variable for extracting the external ip of the instance
+output "ip" {
+ value = google_compute_instance.default.network_interface.0.access_config.0.nat_ip
+}
```
Comment (likamrat) [inline azure_arc_servers_jumpstart/gcp/windows/main.tf:86]: - folder name shows "windows" but this looks like ubuntu deployment.
- I think the name is too long. Also, consider the 15 characters' Windows hostname limit. Can drop the "azure" IMO.

File: azure_arc_servers_jumpstart/gcp/windows/provider.tf [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  features {}
+  subscription_id = var.subscription_id
+  client_id = var.client_id
+  client_secret = var.client_secret
+  tenant_id = var.tenant_id
+}
+
+provider "local" {
+  version = "1.4"
+}
```
Comment (likamrat) [inline azure_arc_servers_jumpstart/gcp/windows/provider.tf:20]: - folder name shows "windows" but this looks like ubuntu deployment.

Comment (youf05) [reply azure_arc_servers_jumpstart/gcp/windows/provider.tf]: removed accidental windows folder commit

File: azure_arc_servers_jumpstart/gcp/windows/main.tf (hunk 2 above)
Comment (youf05) [reply azure_arc_servers_jumpstart/gcp/windows/main.tf]: shortened name and removed windows folder

**Comment by youf05** [comment]
These should be addressed.

File: azure_arc_servers_jumpstart/gcp/ubuntu/terraform/main.tf [hunk 4]
```diff
@@ -11,7 +11,7 @@ resource "azurerm_resource_group" "azure_rg" {
 
 // A single Google Cloud Engine instance
 resource "google_compute_instance" "default" {
- name = "azure-arc-gcp-demo-${random_id.instance_id.hex}"
+ name = "azure-${random_id.instance_id.hex}"
```
Comment (likamrat) [inline azure_arc_servers_jumpstart/gcp/ubuntu/terraform/main.tf:14]: arc-gcp-demo without the "azure"

Comment (youf05) [reply azure_arc_servers_jumpstart/gcp/ubuntu/terraform/main.tf:14]: done

**Pull request merged**